migrator.sh

```diff
@@ -334,7 +334,9 @@ docker_login() {
     # keep retrying docker login until successful
     while [ "$LOGIN_SUCCESS" = "false" ]
     do
-      docker login ${REGISTRY} && LOGIN_SUCCESS="true"
+      read -p "Username: " USERNAME
```

Comment:
I was keeping the existing workflow intact (i.e. prompting for username and password if you don't supply env vars). If you want to eliminate the option to provide a username and password via prompts that would be outside the scope of this merge request.